ScoutSuite/frontend/src/partials/aws/ec2.regions.id.vpcs.id.security_groups.id/RulesList/index.js

```diff
@@ -11,49 +11,45 @@ import './style.scss';
 const RulesList = () => {
   const ctx = useContext(PartialContext);
   const basePath = useContext(PartialPathContext);
-  const value = get(ctx.item, basePath);
+  const rules = get(ctx.item, basePath);
 
   const isDefault = get(ctx.item, 'name') === 'default';
 
-  const renderIpAddresses = (addresses, path) => (
+  const renderIpAddresses = (title, addresses, path) => (
     <li>
-      IP addresses:
+      {`${title}:`}
       <ul>
         {addresses.map((address, i) => (
           <li key={i}>
             <PartialValue
-              value={address.CIDR}
+              value={address}
               errorPath={path}
+              renderValue={value =>
+                value.CIDRName 
+                  ? `${value.CIDR} (${value.CIDRName})`
+                  : value.CIDR
+              }
             />
           </li>
         ))}
       </ul>
     </li>
   );
 
-  const renderIpv6Addresses = addresses => (
-    <li>
-      IPv6 addresses:
-      <ul>
-        {addresses.map((address, i) => (
-          <li key={i}>
-            {/* TODO: actual value for IPv6*/}
-            {address}
-          </li>
-        ))}
-      </ul>
-    </li>
-  );
-
   const renderSecurityGroups = (groups, path) => (
     <li>
       EC2 security groups:
       <ul>
         {groups.map((group, i) => (
           <li key={i}>
             <PartialValue
-              value={`${group.GroupName}\xa0\xa0(${group.GroupId})`}
+              value={group}
               errorPath={path}
+              renderValue={value =>
+                value.GroupName
+                  ? `${value.GroupName} (${value.GroupId})`   //TODO: Link groupId resource
```

Comment:
Do you have task to complete this?

Reply:
yeah but i intended to do all the links in 1 go